### PR TITLE
Operations will now be executed in batches of 500 (specified by Gijs …

### PR DESCRIPTION
**Pull request by mathieub93**
Operations will now be executed in batches of 500 (specified by Gijs as initial setting)

Tested through WeaverNode with operation arrays up to 9 and batches of 1-4.
No additional tests written

### CODE REVIEW

File: src/CoreManager.coffee [hunk 1]
```diff
@@ -65,9 +66,14 @@ class CoreManager
       localTime - serverTime
     )
 
-
-  executeOperations: (operations, target) ->
-    @POST('write', {operations}, target)
+  executeOperations: (allOperations, target) ->
```
Comment (malamili) [inline src/CoreManager.coffee]: Can you try this code instead with your tests? More concise, no code duplication and better readable / understandable (no if statement, no recursion),

```
Promise.mapSeries(_.chunk(allOperations, @maxBatchSize), (operations) =>
     @POST('write', {operations}, target)
)
```

Comment (gvdent) [reply src/CoreManager.coffee]: One thing: does this new code maintain the order of operations over multiple batches? E.g. if batch1 creates a node, and in batch2 a relations is made to that node, it is guaranteed that batch1 is always executed before batch2?

Comment (malamili) [reply src/CoreManager.coffee]: Yes it is guaranteed, _.chunk works in order of elements and mapSeries ensures the promise is resolved before executing the next batch

Comment (mathieub93) [reply src/CoreManager.coffee]: This code works as intended, new PR will be made in a minute

**Pull request merged**